### ISSUE TITLE
Please document the default values of the options

### ISSUE THREAD

**Issue by ChristianCiach** [zmartzone/mod_oauth2#8]
I am trying to migrate from mod_auth_openidc, but I find the documentation inside the conf-file very confusing and lackluster.

For example, I am trying to find out which of the options like `ssl_verify` and `verify.exp` I actually need to specify. Unfortunately, their default values don't seem to be documented.

**Comment by zandbelt** [comment]
yes, documentation is a work in progress and the sample config file is the only source right now

**Comment by fedde-s** [comment]
I'll assume that the link from the README to a wiki that doesn't seem to be there also isn't an unknown issue, then!

(sorry for hijacking this issue report if it is)
